common/serde_utils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "serde_utils"
```

Comment:
We have an external package ([`ethereum_serde_utils`](https://github.com/sigp/ethereum_serde_utils)) which defines a crate called [`serde_utils`](https://docs.rs/ethereum_serde_utils). I think we could either put this code there, give it a different crate name to avoid confusion, or put the `Instant` serialisation as a module in the `lighthouse_network` crate.

I think my preference is for the latter, as the `Instant` stuff is not _really_ Ethereum-specific, and seems a bit too hacky (using `SystemTime::now`) to put in `ethereum_serde_utils`. Similarly, for future serde utils, if we deem them Ethereum-relevant we can put them in the extern crate, whereas if they're Lighthouse-specific we can keep them in-tree. Wdyt?

Reply:
My preference is the former, although I understand your point about lack of Ethereum specificity (and concur). My issue with moving it to `lighthouse_network` is that, in my opinion, that crate is already too busy and this busyness complicates downstream use of the crate *outside* of Lighthouse. I already have some changes that I need to open a (separate) PR for that address some of this.

> and seems a bit too hacky

I also agree here, but my desire for full client-side implementation of the HTTP API overrides this. This certainly isn't a nice solution but my feeling at the moment is that having missing routes is less nice than even this.

Reply:
Oh wow, hadn't considered that WASM quirk before!